don't use GNU extensions in (what should be) highly portable code

--- python-wrapper.c
+++ python-wrapper.c
@@ -170,8 +170,10 @@
 	return ret;
 }
 
-int main(__attribute__((unused)) int argc, char** argv)
+int main(int argc, char** argv)
 {
+	(void)argc; /* pacify compiler */
+
 	if (strlen(program_description) == 0)
 		abort();
 
